<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>1009</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="4" column="0" colspan="18">
     <widget class="QFrame" name="Expenses">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="midLineWidth">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="3">
     <widget class="QLabel" name="Counter">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0 / 12</string>
      </property>
     </widget>
    </item>
    <item row="5" column="16" colspan="2">
     <widget class="QPushButton" name="save_exp">
      <property name="text">
       <string>Save Expenses</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Net Income:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="9" colspan="3">
     <widget class="QLineEdit" name="AnnualRevenue">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Unavailable</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="month_name">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Month</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="5" colspan="2">
     <widget class="QLineEdit" name="Net">
      <property name="placeholderText">
       <string>Press Calculate</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="3">
     <widget class="QLabel" name="NetMonth">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Month</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <widget class="QLineEdit" name="Gross">
      <property name="placeholderText">
       <string>Unavailable</string>
      </property>
     </widget>
    </item>
    <item row="2" column="15" colspan="2">
     <widget class="QLabel" name="UpdatedDate">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="17">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Save as PDF</string>
      </property>
     </widget>
    </item>
    <item row="6" column="7" colspan="3">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Net Income:   Annual</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2" colspan="14">
     <widget class="QLabel" name="Unit_Name">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Unit Data</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Expenses</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6" colspan="3">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Annual Gross Income</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="5">
     <widget class="QLineEdit" name="ExpenseError"/>
    </item>
    <item row="3" column="8" colspan="10">
     <widget class="QLineEdit" name="ErrorEdit"/>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Est. Gross Income</string>
      </property>
     </widget>
    </item>
    <item row="2" column="14">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Updated:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="10" column="5" colspan="2">
     <widget class="QLineEdit" name="Rating">
      <property name="placeholderText">
       <string>Unavailable</string>
      </property>
     </widget>
    </item>
    <item row="8" column="8" colspan="3">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Listing Page for:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="18">
     <widget class="QTextEdit" name="SumNotes">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>250</height>
       </size>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="autoFormatting">
       <set>QTextEdit::AutoNone</set>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="5">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Occupancy Rate</string>
      </property>
     </widget>
    </item>
    <item row="9" column="5" colspan="2">
     <widget class="QLineEdit" name="Occupancy">
      <property name="placeholderText">
       <string>Unavailable</string>
      </property>
     </widget>
    </item>
    <item row="6" column="10" colspan="2">
     <widget class="QLineEdit" name="AnnualNet">
      <property name="placeholderText">
       <string>Press Calculate</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="5">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Avg. Review Rating</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5" colspan="2">
     <widget class="QLineEdit" name="Given">
      <property name="placeholderText">
       <string>Unavailable</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Other Info</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Given Rate</string>
      </property>
     </widget>
    </item>
    <item row="11" column="17">
     <widget class="QPushButton" name="SaveNotes">
      <property name="text">
       <string>Save Notes</string>
      </property>
     </widget>
    </item>
    <item row="8" column="11" colspan="7">
     <widget class="QLabel" name="HyperLink">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="2" column="13">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
